Assets/Scripts/Steam/SteamManager.cs

```diff
@@ -411,15 +423,29 @@ public void StartConnectLoginWithSteamAppTicket(EOSManager.OnConnectLoginCallbac
 #if DISABLESTEAMWORKS
             onLoginCallback?.Invoke(new Epic.OnlineServices.Connect.LoginCallbackInfo() { ResultCode = Epic.OnlineServices.Result.UnexpectedError });
 #else
+            // If this method is called and the login fails, we need to clean out the cached encryptedAppTicket.
+            EOSManager.OnConnectLoginCallback cacheInvalidationOnFailureCallback = (Epic.OnlineServices.Connect.LoginCallbackInfo callbackInfo) => 
```

Comment:
I'll make an informational post about this in the channel and ask for engineer's opinion on this.

The following is my own personal opinion: I never want to use "var". I find it difficult to understand what's being returned, and whenever var is used I have to rely on Visual Studio to guess at what it is. It makes code review harder and glancing over the code is more difficult, because I have to re-contextualize what the variable each time I look at it.

Reply:
I understand your perspective. Good call to poll the engineering team as a whole.